cloudinit/sources/DataSourceVMware.py

```diff
@@ -292,6 +344,543 @@ def redact_keys(self):
         if self.data_access_method == DATA_ACCESS_METHOD_GUESTINFO:
             guestinfo_redact_keys(keys_to_redact, self.vmware_rpctool)
 
+    def get_envvar_data_fn(self):
```

Comment:
Hmm, why the `_fn` suffix on these functions? None of the other functions do this. I recommend removing `_fn` from the new function names.

Reply:
Here I want to make sure each real get_data_fn() ends with `_fn`, so we can tell this is a data transport get data function from its name.